### PR TITLE
Drop old versions of php/sf

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is BC.

## Changelog

```markdown
### Removed
- Support for old versions of PHP and Symfony.
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
         "symfony-cmf/resource": "^1.0",
         "symfony-cmf/resource-bundle": "^1.0",
         "symfony-cmf/testing": "^2.0",
-        "symfony/phpunit-bridge": "^2.8 || ^3.0",
-        "symfony/security-acl": "^2.8 || ^3.0"
+        "symfony/phpunit-bridge": "^3.3",
+        "symfony/security-acl": "^2.8 || ^3.2"
```
Comment (jordisala1991) [inline composer.json]: This should be ^3.0

Comment (core23) [reply composer.json]: Fixed

Comment (jordisala1991) [reply composer.json]: it is still 3.0

Comment (core23) [reply composer.json]: Looks like a github Problem oO
https://github.com/core23/SonataDoctrinePhpcrAdminBundle/blob/patch/php-support/composer.json#L40

**Comment by jordisala1991** [comment]
It does not let me reopen it :S

**Pull request closed without merging**